### PR TITLE
Added hover effect for the service mesh strength comparison section

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Only merge if we intend to keep this one over the current, or ignore 🙂

**Comment by leecalcote** [comment]
At the end of the transition, there is a slight bump to the left for the comparative text. Weird.

**Comment by leecalcote** [comment]
That slight bump at the end doesn't happen in Safari, but does in Chrome.

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
@dhruv0000, please have a look at the SMI table. I tossed in a couple updates. They will be visible on https://layer5ng.netlify.app/projects/landscape soon.

File: src/sections/Landscape-grid/index.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ const LandscapeGrid=() => {
                                 <div className="comparison-mesh">
                                     <div className="mesh-img">
                                         <img src={Istio_Img} />
-                                        <h5>Istio</h5>
+                                        <h2>Istio</h2>
```
Comment (Nikhil-Ladha) [inline src/sections/Landscape-grid/index.js:45]: I used h5, as it went with the front-size mentioned in the brand guide.
As per the brand guide h2 should have a font-size of 2.125 rem.

Comment (leecalcote) [reply src/sections/Landscape-grid/index.js:45]: Oh, that makes sense. The font weight was in need of being adjusted, but as I went to make that adjustment, I arbitrarily and inappropriately changed to h2. I wonder which heading is the best-fit size for these.

Comment (Nikhil-Ladha) [reply src/sections/Landscape-grid/index.js:45]: Ohh, I think the h5 heading goes well. Since it's a caption/name. I will update.

**Pull request merged**